content/en/blog/_posts/2019-06-29-some-apis-are-being-deprecated.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: blog
+title: 'Some APIs Are Being Deprecated: Here's What You Need To Know
+date: 2019-06-29
+---
+
+**Author**: Vallery Lancey (Lyft)
+
+As part of evolving the Kubernetes API, APIs are periodically
+re-organized or upgraded. When this happens, the old API is deprecated, and then
+eventually removed.
+
+There are **4** sets of resources with APIs that are deprecated, will be stop
+being served in the coming months. If you use any of these, this post is for you:
+
+* NetworkPolicy
+* PodSecurityPolicy
+* DaemonSet, Deployment, StatefulSet, and ReplicaSet
+* Ingress
+
+None of these resources are being removed from Kubernetes. NetworkPolicies,
+Deployments, and so on will still remain part of Kubernetes, and are not being
+deprecated in any way. However, to continue using these resources, you must
+use a current version of the Kubernetes API.
+
+# Migration Details
+
+* NetworkPolicy: will no longer be served from **extensions/v1beta1** in **v1.16**.
```

Comment:
Replace with:
```
## Migration details for v1.16

**NetworkPolicy** will no longer be served from `extensions/v1beta1`. To continue using NetworkPolicy, you must migrate to the `networking.k89s.io/v1` API, available since v1.8, where you can retrieve existing persisted data.

**PodSecurityPolicy** will no longer be served form `extensions/v1beta1` in v1.16. To continue using PodSecurityPolicy, you must migrate to the `policy/v1beta1` API, available since v1.10, where you can retrieve existing persisted data.

**DaemonSet**, **Deployment**, **StatefulSet**, and **ReplicaSet** will no longer be served from `extensions/v1beta1`, `apps/v1beta1`, or `apps/v1beta2`. To continue using these services, you must migrate to the `apps/v1` API, available since v1.9, where you can retrieve existing persisted data.

## Migration details for v1.18

**Ingress* will no longer be served from `extensions/v1beta1`. To continue using Ingress, you must migrate to the `networking.k8s.io/v1beta1` API, where you can retrieve existing persisted data.
```